From abf8c4bf53a84ef3de530519a3bbb6b599595f2c Mon Sep 17 00:00:00 2001
From: robdewit <rdewit@wise-guys.nl>
Date: Tue, 31 Mar 2015 14:24:15 +0200
Subject: [PATCH 7/9] Update main.c

Removed bug where cron.update is not picked up while jobs are still running. (We have long running cronjobs and experienced long overdue cron.update files)
---
 main.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/main.c b/main.c
index 595f07a..0eba280 100644
--- a/main.c
+++ b/main.c
@@ -338,7 +338,8 @@ main(int ac, char **av)
 					SynchronizeDir(SCDir, "root", 0);
 					ReadTimestamps(NULL);
 				}
-			} else {
+			} 
+			if (rescan < 60) {
 				CheckUpdates(CDir, NULL, t1, t2);
 				CheckUpdates(SCDir, "root", t1, t2);
 			}
-- 
2.13.2

